src/main/java/uc/user/register/URegisterResponseModel.java

```diff
@@ -19,6 +19,14 @@ public URegisterResponseModel(User user,String loginStatus, String timestamp) {
         this.timestamp = timestamp;
     }
 
+    /** Get the User entity
+     *
+     * @return returns the User entity related to the user being registered
+     */
+    public User getUser() {
+        return user;
```

Comment:
I'm not 100% about this but I'm worried that we may not be able to include entities in the response model as that would require that the presenter depends on entities. Milestone 4 feedback criticised the controllers depending on entities so I think it is a similar principle. We can update this at a later time though - it may just be a matter of representing the useful information about an entity directly within the response model. The difficulty will be when we have to return multiple entities, e.g. a list of courses. Maybe using some sort of model interface in the interface adapters layer would work

Reply:
I agree, I believe it violates clean architecture if we have the entity in the response model.

Reply:
I'll merge with its inclusion for now as we have to revisit all of our response models anyway. @darlynnguyen let me know if you have any ideas how to get around the general problem as mentioned on Slack